### ISSUE TITLE
Prismarine Crystal Particle - No options

### ISSUE THREAD

**Issue by xanthian** [Ladysnake/Illuminations#109]
![image](https://user-images.githubusercontent.com/7688001/145657768-cca6475e-7193-4a0d-bfd1-f1b74f54d5a0.png)

Please could you add an option to lower the density of the Prismarine Crystal Particles, my underwater home is covered in Sea Lanterns and this is having a significant impact on my FPS (profiled using Spark this was the highest impacting)

I've tried disabling Plankton in case it was this, but it has no impact.

Illuminations-1.10.1
MC 1.17.1
Fabric

**Comment by doctor4t** [comment]
Holy moly I'll try to do something

**Comment by griffin4cats** [comment]
Yeah, this is really bad. I reported it on the discord a bit ago, i should've checked for a github issue though

**Comment by xanthian** [comment]
any update on this?
